crates/gitbutler-operating-modes/src/lib.rs

```diff
@@ -1,38 +1,113 @@
+use std::{fs, path::PathBuf};
+
 use anyhow::{bail, Context, Result};
 use gitbutler_command_context::CommandContext;
+use serde::{Deserialize, Serialize};
+
+pub const INTEGRATION_BRANCH_REF: &str = "refs/heads/gitbutler/integration";
```

Comment:
why is this defined here? There are other places in the app that use this string, perhaps it should live there